flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/Table.java

```diff
@@ -879,4 +879,117 @@
 	 * @return An OverWindowedTable to specify the aggregations.
 	 */
 	OverWindowedTable window(OverWindow... overWindows);
+
+	/**
+	 * Performs a field add operation. Similar to an SQL SELECT statement. The field expressions
+	 * can contain complex expressions, but can not be an aggregations. Will report an expression if
```

Comment:
- can not be an aggregations => can not contain aggregations
- report an expression => report an exception